docs/preprocess-templates/synchrotron_beta_logpoltens.ipynb

```diff
@@ -0,0 +1,325 @@
+   "id": "8739464f-3c93-48dd-a59e-15f7f85a0c76",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "template_dir = \"/global/project/projectdirs/cmb/www/pysm-data/pysm_2/\"\n",
+    "beta_s_pysm = hp.read_map(template_dir + \"synch_beta.fits\")\n",
+    "print(beta_s_pysm.mean())\n",
+    "beta_s_pysm_nomono = hp.remove_monopole(beta_s_pysm)\n",
+    "print(beta_s_pysm_nomono.mean())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56bae391-7618-44cc-945e-fbf27023e9fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ell = np.arange(lmax + 1)\n",
+    "cl_spass_fit = 10 ** (-0.82655497) * ell ** (-2.60860208)\n",
+    "cl_spass_fit[0] = 0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "736969cf-d2c3-4361-972f-41b5d87a99e7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pysm3.utils import sigmoid"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4e0b6047-4011-4fe5-b1f3-08f10951266b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ell_fit_low = {\"TT\": 10, \"EE\": 10, \"BB\": 10}\n",
+    "ell_fit_high = {\"TT\": 36, \"EE\": 36, \"BB\": 36}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d805b7cb-c40b-4fb3-83bf-a055bc6d4a35",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.random.seed(444)\n",
+    "\n",
+    "bs_ss_alm = hp.synalm(cl_spass_fit, lmax=output_lmax)\n",
+    "alm_bs = hp.map2alm(beta_s_pysm_nomono, lmax=output_lmax, use_pixel_weights=True)\n",
+    "\n",
+    "bs_LS_alm = np.empty_like(alm_bs)\n",
+    "\n",
+    "sig_func = sigmoid(ell, x0=ell_fit_high[\"TT\"] - 15, width=ell_fit_high[\"TT\"] / 10)\n",
+    "bs_LS_alm = hp.almxfl(alm_bs, np.sqrt(1.0 - sig_func))\n",
+    "bs_ss_alm = hp.almxfl(bs_ss_alm, np.sqrt(sig_func))\n",
+    "bs_ss = hp.alm2map(bs_ss_alm, nside=output_nside)\n",
+    "\n",
+    "bs_ss *= modulate_amp\n",
+    "\n",
+    "bs_ls = hp.alm2map(bs_LS_alm, nside=output_nside)\n",
+    "bs_ls_resc = bs_ls * 1.5722501"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "059293d5-95bd-43ae-b8df-1226a68d893c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hp.write_cl(\n",
+    "    output_dir / \"raw\" / f\"synch_small_scales_beta_cl_lmax{output_lmax}.fits\",\n",
+    "    cl_spass_fit,\n",
+    "    dtype=np.complex128,\n",
+    "    overwrite=True,\n",
+    ")\n",
+    "\n",
+    "hp.write_alm(\n",
+    "    output_dir / \"raw\" / f\"synch_largescale_beta_alm_nside{nside}_lmax{lmax:d}_complex64.fits\",\n",
+    "    bs_LS_alm,\n",
+    "    lmax=lmax,\n",
+    "    out_dtype=np.complex64,\n",
+    "    overwrite=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "210409a6-f7b9-451a-9f8b-da50dd8c9f89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bs_out = bs_ss * modulate_amp + bs_ls_resc - 3.1"
```

Comment:
@giuspugl and here, I'll remove one of them, please let me know if instead I should leave it.

Reply:
yeah that seems a bug! 
sorry about that!